helmfile.d/values/networkpolicies/common/cert-manager.yaml.gotmpl

```
@@ -31,6 +31,15 @@ policies:
           ports:
             - tcp: 443
         {{- end }}
+        {{- with .Values | get "networkPolicies.certManager.dns01.ips" list }}
+        - name: dns01
```

Maybe we should follow the convention (or change the convention).
```suggestion
        - name: egress-rule-dns01
```

---

No, sounds good to change it to your suggestion.